src/main/java/slimeknights/tconstruct/library/client/GuiUtil.java

```diff
@@ -153,6 +153,22 @@ public static void drawGuiTank(SmelteryTank liquids, int x, int y, int w, int he
     }
   }
 
+  public static FluidStack getFluidStackAtPosition(SmelteryTank tank, int mouseX, int mouseY, int xmin, int ymin, int xmax, int ymax) {
```

Comment:
Nearly, the main difference is the code in the exactiddle changed, and I was not sure of the cleanest way to do that.